pkg/processor/build/builder.go

```diff
@@ -476,6 +480,16 @@ func (b *Builder) copyObjectsToStagingDir() error {
 
 	b.logger.DebugWith("Runtime provided objects to staging dir", "objects", objectPathsToStagingDir)
 
+	filesToCopy := b.processorImage.filesToCopy
+	if b.processorImage.scriptPathToRunDuringBuild != "" {
+		filesToCopy = append(filesToCopy, b.processorImage.scriptPathToRunDuringBuild)
+	}
+
+	for _, filePath := range filesToCopy {
+		basePath := filepath.Base(filePath)
+		objectPathsToStagingDir[filepath.Join(b.GetFunctionDir(), basePath)] = "/opt/nuclio"
```

Comment:
Why are we not using `path.Join` here?

Reply:
For `/opt/nuclio`? It's on a Docker container so it'll be a POSIX file system in any case.